sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/src/Microsoft.Azure.Management.Sql.csproj

```diff
@@ -7,7 +7,7 @@
 		<PackageId>Microsoft.Azure.Management.Sql</PackageId>
 		<Description>Azure SQL Management SDK library</Description>
 		<AssemblyName>Microsoft.Azure.Management.Sql</AssemblyName>
-		<Version>1.48.0-preview</Version>
+		<Version>1.49.0-preview</Version>
```

Comment:
Was this on purpose? Same goes for the file below.

Reply:
Yes, I believe version should be updated with changes

Reply:
Same doubt here, is there any reason to update versions with just tests updates?

Reply:
You're right. I will revert this. I thought that it should be updated with every change.